src/pages/index.astro

```diff
@@ -65,6 +65,9 @@
 		var supabase = createClient(SUPABASE_URL, SUPABASE_KEY)
 		window.userToken = null
 
+		console.log(supabase.auth.user())
+		console.log(supabase.auth.session())
```

Comment:
note that we're about to deprecate these for `getUser()` and `getSession()` (both async)